### PR TITLE
Use a lite version of @repeater/repeaterjs

### PR DESCRIPTION
**Pull request by maneetgoyal**
## PR Checklist

- [x] Important or complicated code is tested

```

Test Suites: 44 passed, 44 total
Tests:       446 passed, 446 total
Snapshots:   204 passed, 204 total
Time:        6.371 s
```

Live Testing:

https://github.com/gadget-inc/js-clients/assets/25290212/13091fa7-7606-4039-8107-16ecf7e073a3

- [ ] Any user facing changes are documented in the Gadget-side changelog
- [ ] Any immediate changes are slated for release in Gadget via a generated package dependency bump
- [ ] Versions within this monorepo are matching and there's a valid upgrade path

### CODE REVIEW

**Comment by maneetgoyal** [comment]
@airhorns Can you please take a look?

File: packages/api-client-core/src/graphql-live-query-utils/index.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { createApplyLiveQueryPatch } from "./createApplyLivequeryPatch.js";
+import { applyJSONDiffPatch } from "@n1ru4l/graphql-live-query-patch-jsondiffpatch";
+
+export const applyLiveQueryJSONDiffPatch =
+    createApplyLiveQueryPatch(applyJSONDiffPatch);
```
Comment (airhorns) [inline packages/api-client-core/src/graphql-live-query-utils/index.ts:5]: Can we import some of the tests from the original package and test the imported code here?

File: pnpm-lock.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```
Comment (airhorns) [inline pnpm-lock.yaml:1]: Would you mind re-running with an up to date pnpm version so this doesn't regress?

File: packages/api-client-core/package.json [hunk 3]
```diff
@@ -28,6 +28,7 @@
     "prerelease": "gitpkg publish"
   },
   "dependencies": {
+    "@manufac/repeater": "^4.0.6",
```
Comment (airhorns) [inline packages/api-client-core/package.json:31]: Can you share the source for this package with us? I'm fine either way if you want to keep it under your github account or move it to ours, just want to make sure we can patch it further once the engagement is done

**Review by airhorns — COMMENTED** [review]
This is the

**Comment by airhorns** [comment]
I think if you rebase or merge main, you should get github actions running 👍

File: packages/api-client-core/package.json (hunk 3 above)
Comment (maneetgoyal) [reply packages/api-client-core/package.json:31]: Added you to the repo. Planning to add all that code into this same monorepo in a subsequent PRs. They are using a different version of TS which was causing some compilation errors with the TS config used here.

**Pull request closed without merging**